File: fault.py
class Fault:
    def __init__(self, pos, CFF):
        self._pos = pos
        self._CFF = CFF
        self._r = CFF._r

    def Apply(self, heightmap):
        self._CFF.Apply(heightmap, self.GetX(), self.GetY())

    def GetX(self):
        return int(self._pos[0])

    def GetY(self):
        return int(self._pos[1])


    def __str__(self):
        return "(" + str(self._pos[0]) + "," + str(self._pos[1]) + ", r = " + str(self._r) + ")"

File: C#/Generator/Fault.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MesaGen.Generator
{
    public class Fault
    {
        (int x, int y) pos;
        CFFKernel CFF;
        int r;

        public Fault((int x, int y) pos, CFFKernel CFF) {
            this.pos = pos;
            this.CFF = CFF;
            r = CFF.r;
        }

        public void Apply(Heightmap heightmap) {
            CFF.Apply(heightmap, GetX(), GetY());
        }

        public int GetX() {
            return pos.x;
        }

        public int GetY() {
            return pos.y;
        }

        public override string ToString() {
            return $"({pos.x}, {pos.y}, r = {r})";
        }
    }
}
